server/example_test.go

```diff
@@ -1,14 +1,15 @@
 package server
 
 import (
+	"crypto"
+	"fmt"
 	"log"
 
 	"github.com/google/go-tpm-tools/client"
 	"github.com/google/go-tpm-tools/simulator"
-	"github.com/google/go-tpm/tpm2"
 )
 
-func ExampleParseMachineState() {
+func ExampleVerifyAttestation() {
 	// On client machine, generate the TPM quote.
```

Comment:
I wonder if it makes sense to keep this test around given that we already have the Attest example. The only reason I can think of is for clear examples for both `server.VerifyAttestation` and `client.Attest`, using many of the available opts.

Just some bikeshedding here. Either way, don't need to worry about it in this PR.

Reply:
ya, seems like it is a little bit repetitive. Problem is that these two tests are touching both client and server. We can find a better place one of them later